cicd/forgeops-tests/configure.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 ForgeRock AS. Use of this source code is subject to the
+# Common Development and Distribution License (CDDL) that can be found in the LICENSE file
+
+"""
+Customize the helm charts (change image tag, repository,...) and list of components to deploy
+"""
+
+# Python imports
+import sys
+import os
+import shutil
+import yaml
+import re
+
+TESTS_NAMESPACE = "smoke"
+if "TESTS_NAMESPACE" in os.environ:
+    TESTS_NAMESPACE = os.environ["TESTS_NAMESPACE"]
+
+TESTS_DOMAIN = "forgeops.com"
+if "TESTS_DOMAIN" in os.environ:
+    TESTS_DOMAIN = os.environ["TESTS_DOMAIN"]
+
+TESTS_IMAGE_TAG = "7.0.0-SNAPSHOT"
+if "TESTS_IMAGE_TAG" in os.environ:
+    TESTS_IMAGE_TAG = os.environ["TESTS_IMAGE_TAG"]
+
+TESTS_DEPLOYMENT = "smoke-deployment"
+if "TESTS_DEPLOYMENT" in os.environ:
+    TESTS_DEPLOYMENT = os.environ["TESTS_DEPLOYMENT"]
+
+TESTS_COMPONENTS = ""
+if "TESTS_COMPONENTS" in os.environ:
+    TESTS_COMPONENTS = os.environ["TESTS_COMPONENTS"]
+
+IMAGE_BASE_URL = 'forgerock-docker-public.bintray.io/forgerock'
+IMAGE_PULL_POLICY = 'Always'
+
+OPENAM_HELM_SUBFOLDER = "openam"
+OPENAM_IMAGE_NAME = "openam"
+OPENAM_DEPENDENCIES = "configstore"
+
+AMSTER_HELM_SUBFOLDER = "amster"
+AMSTER_IMAGE_NAME = "amster"
+AMSTER_DEPENDENCIES = ""
+
+USERSTORE_HELM_SUBFOLDER = "ds"
+USERSTORE_IMAGE_NAME = "ds"
+USERSTORE_DEPENDENCIES = ""
+
+CTSSTORE_HELM_SUBFOLDER = "ds"
+CTSSTORE_IMAGE_NAME = "ds"
+CTSSTORE_DEPENDENCIES = ""
+
+CONFIGSTORE_HELM_SUBFOLDER = "ds"
+CONFIGSTORE_IMAGE_NAME = "ds"
+CONFIGSTORE_DEPENDENCIES = ""
+
+OPENIDM_HELM_SUBFOLDER = "openidm"
+OPENIDM_IMAGE_NAME = "idm"
+OPENIDM_DEPENDENCIES = "postgres-openidm"
+
+OPENIG_HELM_SUBFOLDER = "openig"
+OPENIG_IMAGE_NAME = "ig"
+OPENIG_DEPENDENCIES = ""
+
+if __name__ == "__main__":
+    # root_dir : computed using relative position from this file
+    current_dir = os.path.abspath(os.path.dirname(__file__))
+    os.chdir(os.path.join(os.path.abspath(os.path.dirname(__file__)), '..', '..'))
+    root_dir = os.getcwd()
+    os.chdir(current_dir)
```

Comment:
Are you sure we want to change the current working directory after computing root__dir?

Reply:
To have full path instead of having path with `../../`

Reply:
Guillaume helped me to find a better solution